test/vercel/UpdateProject.test.js

```
@@ -0,0 +1,164 @@
+                        name: 'new-name'
+                        // Other fields are undefined/null
+                    }
+                }
+            },
+            auth: {
+                apiToken: process.env.VERCEL_API_TOKEN
+            },
+            httpRequest: async (options) => {
+                assert(options.data);
+                assert.strictEqual(options.data.name, 'new-name');
+                assert(!options.data.hasOwnProperty('devCommand'));
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)